app/furniture/marketplace/locales/en.yml

```diff
@@ -37,6 +37,10 @@ en:
       manage_header: "Marketplace"
       configuration_menu: "Configuration"
       edit: "Manage Marketplace"
+      config_missing_title: "Marketplace configuration missing"
+      config_missing_explainer: |
+        This marketplace is not ready for shopping, and is not visible to the public.
+        Add missing configuration by going to 'Manage Marketplace'.
```

Comment:
Any chance we could make `Manage Marketplace` a link? I don't know how i18n handles html tho; but I imagine something like`t("..", edit_link: link_to(t('marketplace.marketplaces.edit.link_to'), marketplace.location(:edit))` may work?

Reply:
Done.